### ISSUE TITLE
Choose Images after uploaded

### ISSUE THREAD

**Issue by nguyenkhanhhoalk** [JakeHartnell/react-images-upload#167]
Hi dear,
How I choose pictures after i uploaded and get properties of picture such as height, width of picture . . .? 
Thank for support

**Comment by BingyuLiu** [comment]
yes, I saw the onChange event can't be called.  How can I capture the picture data so that I can upload it to my API?
